hack/generate-crds.sh

```diff
@@ -21,7 +21,7 @@ set -o pipefail
 CRD_VERSIONS=v1
 CRD_OUTPUTS=build/crds
 DEVICES_VERSION=v1beta1
-OPERATIONS_VERSION=v1alpha1
+OPERATIONS_VERSION=v1alpha2
```

Comment:
Now we still use v1alpha1 by default?  Will there be a problem with changing this to v1alpha2?

Reply:
My original idea was to gradually transition to v1alpha2, but it seems that there are not many users using node jobs now, so it's ok to use v1alpha2 by default. I just need to adjust the definition of feature-gate. Do we need to use v1alpha2 by default?

Reply:
The node jobs of the v1alph2 have been used as the default.